firefox-ios/Client/RatingPromptManager.swift

```diff
@@ -16,8 +16,8 @@ final class RatingPromptManager {
 
     struct Constants {
         static let minDaysBetweenReviewRequest = 60
-        static let firstThreshold = 30
-        static let secondThreshold = 90
+        static let firstThreshold = 15
+        static let secondThreshold = 40
```

Comment:
Just a thought - Not to increase scope here, but if these values are going to change with the experiment, maybe can add to yaml config.

Reply:
I can ask Andy!

Reply:
He said its fine without 👍